tests/koji/test_koji_vmi.py

```diff
@@ -115,7 +118,7 @@ def test_koji_vmis(fake_koji, koji_dir):
                 "btype": "image",
                 "filename": fname,
                 "checksum": "00000000000000000000000000000000",
-                "extra": {"image": {"arch": "x86_64"}},
+                "extra": {"image": {"arch": "x86_64", "boot_mode": boot_mode}},
```

Comment:
This looks broken/unrealistic.  By this line of code `boot_mode` variable is an object of type BootMode, the 'extra' dict returned by koji will never return an instance of that type, only strings, so the test setup is inaccurate.

I believe the code elsewhere may be buggy and this mistake may be allowing the test to pass regardless when it should really be failing.

Reply:
Thanks! It's fixed now, PTAL again